### ISSUE TITLE
Ghost User still not  fixed 

### ISSUE THREAD

**Issue by Ana-Green** [HalcyonGrid/halcyon#112]
[20:35] Mayumi Miyata is online.nut  she is not online as she lays in the hospital, now again presence show that her UUID is online with some  0000 in the UUID again a  bug. 

Log 👎 

[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:01:12 2020)I was not here these days I was sick I stayed 5 days in the hospital
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:01:21 2020)i in home now
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:00:07 2020)sorry Frank
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:01:46 2020)I confess I'm scared
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:03:56 2020)i love ya
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:03:53 2020)lots pain now I  feel a  bit better
[19:03] Mayumi Miyata: (Saved Sun Sep 06 02:03:35 2020)51/5000
I will take some medicine and wait for the surgery

Regards,
Ana Green

**Comment by life777eternal** [comment]
That's true, I saw that again on [Amaryllis Grid](https://AmaryllisGrid.com) recently at the Casablanca region.

**Comment by Vinhold** [comment]
If a user in a Halcyon world crashes out of the world for whatever reason, their in world "presence" will still remain as logged in until they can logon again and correctly log out. There is no detection in the Halcyon region process for a lost connection that signals to the grid services to log out the account on a timeout.

**Comment by life777eternal** [comment]
[Sovaria Estates](https://SovariaEstates.world) has ghosting again now too. 

<img width="1600" height="877" alt="Image" src="https://github.com/user-attachments/assets/90bf6dae-ac6d-4cda-b495-929d86749640" />

This is a picture I took with two who were already logged out at the time... plus one of them had teleported home before logging out. Aye @Vinhold this would be a duplicate of #100 but that one didn't show up when I searched for "ghost". Shouldn't that kind of thing be noted under "Relationships" at the top right? @kf6kjg 

Thank you Shalom.

**Comment by kf6kjg** [comment]
Closing as duplicate of #100
